tests/cms/models/media/test_media_file.py

```diff
@@ -0,0 +1,248 @@
+
+    @pytest.mark.django_db
+    def test_is_not_deletable_if_used_as_organization_icon(self) -> None:
+        region = Region.objects.create(name="Testregion")
+
+        file = MediaFile.objects.create(
+            file="example.pdf",
+            thumbnail="example_thumbnail.jpg",
+            file_size=1024,
+            type="PDF",
+            name="Example File",
+            alt_text="This is an example file",
+            last_modified=datetime(2024, 4, 11, 10, 30, 0),
+            is_hidden=False,
+        )
+
+        # Organization Icon
+        Organization.objects.create(
+            name="Beispiel-Organisation",
+            slug="beispiel-organisation",
+            website="https://example.com",
+            region=region,
+            icon=file,
+        )
+
+        assert not file.is_deletable
+
+    @pytest.mark.django_db
+    def test_is_not_deletable_if_used_as_within_translation_of_upcoming_event(self) -> None:
+        region = Region.objects.create(name="Testregion")
+
+        file = MediaFile.objects.create(
+            file="bielefeld_stadtmarke.png",
+            thumbnail="bielefeld_stadtmarke.png",
+            file_size=1024,
+            type="PNG",
+            name="Example File",
+            alt_text="This is an example file",
+            last_modified=datetime(2024, 4, 11, 10, 30, 0),
+            is_hidden=False,
+        )
+
+        upcoming_event = Event.objects.create(
+            start=timezone.now() + timedelta(days=1),
+            end=timezone.now() + timedelta(days=2),
+            region=region,
+        )
+
+        german_language = Language.objects.create(slug="de", bcp47_tag="de", native_name="Deutsch", english_name="German", text_direction="ltr", primary_country_code="DE", table_of_contents="Inhaltsverzeichnis")
+
+        EventTranslation.objects.create(event=upcoming_event, language=german_language, content=f'<p><img src="http://localhost:8000/media/{file.file}"></p>')
+
+        find_all_links()
+
+        assert not file.is_deletable
+
+    @pytest.mark.django_db
+    def test_is_deletable_if_used_in_translation_of_past_event(self) -> None:
+        region = Region.objects.create(name="Testregion")
+
+        file = MediaFile.objects.create(
+            file="bielefeld_stadtmarke.png",
+            thumbnail="bielefeld_stadtmarke.png",
+            file_size=1024,
+            type="PNG",
+            name="Example File",
+            alt_text="This is an example file",
+            last_modified=datetime(2024, 4, 11, 10, 30, 0),
+            is_hidden=False,
+        )
+
+        past_event = Event.objects.create(
+            start=timezone.now() - timedelta(days=2),
+            end=timezone.now() - timedelta(days=1),
+            region=region,
+        )
+
+        german_language = Language.objects.create(slug="de", bcp47_tag="de", native_name="Deutsch", english_name="German", text_direction="ltr", primary_country_code="DE", table_of_contents="Inhaltsverzeichnis")
```

Comment:
I guess we are getting an error here because German exists already in test data.
We could either load the fixture and use the german language objects from it or create a brand new language with a slug which is nowhere used.